padertorch/summary/tbx_utils.py

```diff
@@ -109,14 +110,38 @@ def stft_to_image(
                `matplotlib.pyplot.cm.get_cmap` to get the color map. If `None`,
                grayscale is used.
         origin: Origin of the plot. Can be `'upper'` or `'lower'`.
+        visible_dB: How many dezibel are visible in the image.
+                    Note: `paderbox.visualization.plot.stft` uses
+                          `visible_dB == 60` internally. So by default it shows
+                          10 dB more.
 
     Returns:
         Colorized image with shape (color (1 or 3), features, frames)
+
+
+    Small test to see the effect of `visible_dB`:
+
+        >>> visible_dB = 60
+        >>> 10 ** (-visible_dB / 20)
+        0.001
+
+        >>> data = [1, 0.004, 0.003, 0.001_05, 0.001]
+        >>> np.squeeze(stft_to_image(np.array(data)[:, None], color=None))
+        array([255,  10,   0,   0,   0], dtype=uint8)
+
+        >>> np.squeeze(stft_to_image(
+        ...     np.array(data)[:, None], color=None, visible_dB=60))
+        array([255,  51,  40,   1,   0], dtype=uint8)
+
     """
     signal = to_numpy(signal, detach=True)
 
     return spectrogram_to_image(
-        np.abs(signal), batch_first=batch_first, color=color, origin=origin,
+        signal.real ** 2 + signal.imag ** 2,
```

Comment:
What's the difference between `np.abs(signal)` and `signal.real ** 2 + signal.imag ** 2`? I would expect that `np.abs` is the better option.

Reply:
I made two changes here:
 - I changed from `np.abs(signal)` to `np.abs(signal)**2`, because we usually define the spectrogram as the power of the STFT.
 - I changed `np.abs(signal)**2` to `signal.real ** 2 + signal.imag ** 2`:
    - Use the same operation as in paderbox
    - `np.abs` includes a square root operation, this avoids such an operation. If I remember correct, that operation is faster for common shapes that we have.

Reply:
Ok, I didn't notice the missing square root. Then, it makes sense.